pypeman/commands.py

```diff
@@ -179,6 +161,15 @@ def main(debug_asyncio=False, profile=False, cli=False, remote_admin=False):
 
     print("End started tasks...")
 
+    # TODO: Shouldn't we also stop all endpoints?
+    # TODO: and if yes, before or after stopping the channels
+    # Intuitively, as we start endpoints after the channels we should probably stop
```

Comment:
Agree with that !

Reply:
OK. shall I do this in this PR or in another one?

Reply:
As you want. You can do it in this one.

Reply:
done in this PR in commit de509453e9116ba39748cf77b851ae86e2c86167